core/client/app/routes/setup/three.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+    setupController: function (controller) {
```

Comment:
To avoid user's being able to manipulate the URL (not sure if that's actually a concern?) you could add a `beforeModel` check here and redirect if the blog hasn't been created, e.g:

```
beforeModel: function () {
    if (!this.controllerFor('setup').get('blogCreated')) {
        this.transitionTo('setup.two');
    }
}
```